howso/direct/client.py

```diff
@@ -1109,14 +1109,19 @@ def persist_trainee(self, trainee_id: str):
 
         self.howso.persist(trainee_id)
 
-    def _auto_resolve_trainee(self, trainee_id: str):
+    def _resolve_trainee(self, trainee_id: str, **kwargs) -> str:
```

Comment:
Seems to be resolving shouldn't load the trainee if it is found. Probably not for this PR, but can resolving and acquiring remain separate methods?

Reply:
This is the existing behavior. If the Trainee is not in the cache we attempt to acquire it automatically.

Reply:
We could consider pushing this onto the user to ensure their trainee is explicitly acquired first. But not something to change yet.